web/proxy.js

```diff
@@ -1,5 +1,12 @@
 const http = require('http');
 const https = require('https');
+require('dotenv').config();
```

Comment:
I don't think this is doing anything, is it? Typically this would be used with a variable like:

```
const config = require('dotenv').config();
```

Reply:
It is. I've used it like this before.

Reply:
Ah, I see. That's what adds `env` to `process`. I was thinking that `process.env` just existed natively.